### PR TITLE
w_debug: New message handling for development

### PR DESCRIPTION
**Pull request by Kreyren**
i'm using this locally to diagnose my contributions and i believe that using this in winetricks would be beneficial

Signed-off-by: Jacob Hrbek <kreyren@rixotstudio.cz>

### CODE REVIEW

**Comment by Kreyren** [comment]
Do not merge: wrong slash used

File: src/winetricks [hunk 1]
```diff
@@ -262,6 +262,13 @@ w_warn()
     unset _W_timeout
 }
 
+# Output debug message to stdin if debug variable is non-zero
+w_debug()
+{
+    # shellcheck disable=SC2154
+    [ -n "$debug" ] && printf 'DEBUG: %s\n' "$1"
```
Comment (austin987) [inline src/winetricks]: A) debug is wayyy too generic of a name.
B) Convention (https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L135) would be `W_DEBUG`

Comment (austin987) [reply src/winetricks]: Also, please assign $1 to a variable (msg or similar)

Comment (Kreyren) [reply src/winetricks]: > Also, please assign $1 to a variable (msg or similar)

Resolved in: https://github.com/Winetricks/winetricks/commit/95c76986c10da49fcdf9f0f9e9c576e500338787

Comment (Kreyren) [reply src/winetricks]: > A) debug is wayyy too generic of a name.

Is that a bad thing?

Comment (Kreyren) [reply src/winetricks]: > B) Convention (https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L135) would be W_DEBUG

Resolved in https://github.com/Winetricks/winetricks/pull/1339/commits/8d806013f8823469affb69afb807d18090e9ca95

**Comment by Kreyren** [comment]
Synced contrib with kreytricks

**Pull request closed without merging**